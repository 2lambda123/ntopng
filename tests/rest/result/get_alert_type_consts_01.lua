{"rc":0,"rc_str":"OK","rsp":{"alert_am_threshold_cross":{"alert_key":30},"alert_blacklisted_country":{"alert_key":1},"alert_broadcast_domain_too_large":{"alert_key":2},"alert_connection_issues":{"alert_key":56},"alert_device_connection":{"alert_key":3},"alert_device_disconnection":{"alert_key":4},"alert_device_protocol_not_allowed":{"alert_key":5},"alert_dropped_alerts":{"alert_key":6},"alert_flow_blacklisted":{"alert_key":8},"alert_flow_blocked":{"alert_key":9},"alert_flow_misbehaviour":{"alert_key":10},"alert_flows_flood":{"alert_key":11},"alert_ghost_network":{"alert_key":12},"alert_host_log":{"alert_key":59},"alert_host_pool_connection":{"alert_key":13},"alert_host_pool_disconnection":{"alert_key":14},"alert_influxdb_error":{"alert_key":16},"alert_influxdb_export_failure":{"alert_key":17},"alert_internals":{"alert_key":18},"alert_ip_outsite_dhcp_range":{"alert_key":19},"alert_known_proto_on_non_std_port":{"alert_key":58},"alert_list_download_failed":{"alert_key":20},"alert_login_failed":{"alert_key":21},"alert_mac_ip_association_change":{"alert_key":22},"alert_malicious_signature":{"alert_key":23},"alert_misconfigured_app":{"alert_key":25},"alert_new_device":{"alert_key":26},"alert_nfq_flushed":{"alert_key":27},"alert_none":{"alert_key":28},"alert_periodic_activity_not_executed":{"alert_key":29},"alert_port_duplexstatus_change":{"alert_key":31},"alert_port_errors":{"alert_key":32},"alert_port_load_threshold_exceeded":{"alert_key":33},"alert_port_mac_changed":{"alert_key":34},"alert_port_status_change":{"alert_key":35},"alert_potentially_dangerous_protocol":{"alert_key":36},"alert_process_notification":{"alert_key":37},"alert_quota_exceeded":{"alert_key":38},"alert_remote_to_remote":{"alert_key":39},"alert_request_reply_ratio":{"alert_key":40},"alert_slow_periodic_activity":{"alert_key":41},"alert_slow_purge":{"alert_key":42},"alert_snmp_device_reset":{"alert_key":43},"alert_snmp_topology_changed":{"alert_key":44},"alert_suspicious_activity":{"alert_key":45},"alert_suspicious_file_transfer":{"alert_key":57},"alert_tcp_syn_flood":{"alert_key":46},"alert_tcp_syn_scan":{"alert_key":47},"alert_threshold_cross":{"alert_key":49},"alert_too_many_drops":{"alert_key":50},"alert_udp_unidirectional":{"alert_key":51},"alert_user_activity":{"alert_key":53},"alert_user_script_calls_drops":{"alert_key":54},"alert_web_mining":{"alert_key":55},"external_alert":{"alert_key":7}}}
