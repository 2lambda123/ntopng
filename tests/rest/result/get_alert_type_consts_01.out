{"rc":0,"rc_str":"OK","rc_str_hr":"Success","rsp":[{"type":"alert_dns_data_exfiltration"},{"type":"alert_login_failed"},{"type":"alert_blacklisted_country"},{"type":"alert_shell_script_executed"},{"type":"alert_unexpected_ntp_server"},{"type":"alert_flow_new_api_demo"},{"type":"alert_flow_blocked"},{"type":"alert_tcp_syn_flood_victim"},{"type":"alert_nfq_flushed"},{"type":"alert_quota_exceeded"},{"type":"alert_mac_ip_association_change"},{"type":"alert_host_new_api_demo"},{"type":"alert_am_threshold_cross"},{"type":"alert_ghost_network"},{"type":"alert_tcp_connection_refused"},{"type":"alert_fail2ban_executed"},{"type":"alert_tls_certificate_expired"},{"type":"alert_port_duplexstatus_change"},{"type":"alert_ip_outsite_dhcp_range"},{"type":"alert_suspicious_tcp_probing"},{"type":"alert_user_activity"},{"type":"alert_longlived"},{"type":"alert_tls_old_protocol_version"},{"type":"alert_host_log"},{"type":"alert_process_notification"},{"type":"alert_zero_tcp_window"},{"type":"alert_tls_certificate_mismatch"},{"type":"alert_influxdb_error"},{"type":"alert_suspicious_file_transfer"},{"type":"alert_web_mining"},{"type":"alert_dns_invalid_query"},{"type":"alert_list_download_failed"},{"type":"alert_iec104_error"},{"type":"alert_malicious_signature"},{"type":"alert_host_pool_disconnection"},{"type":"alert_slow_periodic_activity"},{"type":"alert_data_exfiltration"},{"type":"alert_unexpected_dhcp_server"},{"type":"alert_port_status_change"},{"type":"alert_udp_unidirectional"},{"type":"alert_slow_purge"},{"type":"alert_unexpected_new_device"},{"type":"alert_port_load_threshold_exceeded"},{"type":"alert_host_pool_connection"},{"type":"alert_flow_low_goodput"},{"type":"alert_tcp_syn_scan_victim"},{"type":"alert_periodic_activity_not_executed"},{"type":"alert_suspicious_tcp_syn_probing"},{"type":"alert_too_many_drops"},{"type":"alert_tcp_syn_scan_attacker"},{"type":"alert_tls_certificate_selfsigned"},{"type":"alert_known_proto_on_non_std_port"},{"type":"alert_flow_risk"},{"type":"alert_request_reply_ratio"},{"type":"alert_misconfigured_app"},{"type":"alert_unexpected_dns_server"},{"type":"alert_port_errors"},{"type":"alert_threshold_cross"},{"type":"alert_device_connection"},{"type":"alert_periodicity_update"},{"type":"alert_external"},{"type":"alert_broadcast_domain_too_large"},{"type":"alert_influxdb_export_failure"},{"type":"alert_snmp_device_reset"},{"type":"alert_snmp_topology_changed"},{"type":"alert_potentially_dangerous_protocol"},{"type":"alert_contacted_peers"},{"type":"alert_device_disconnection"},{"type":"alert_tls_unsafe_ciphers"},{"type":"alert_remote_to_remote"},{"type":"alert_internals"},{"type":"alert_flows_flood_attacker"},{"type":"alert_flow_blacklisted"},{"type":"alert_elephant_local_to_remote"},{"type":"alert_flows_flood_victim"},{"type":"alert_iec_invalid_transition"},{"type":"alert_port_mac_changed"},{"type":"alert_no_if_activity"},{"type":"alert_user_script_calls_drops"},{"type":"alert_dns_positive_error_ratio"},{"type":"alert_lateral_movement"},{"type":"alert_attack_mitigation_via_snmp"},{"type":"alert_connection_issues"},{"type":"alert_device_protocol_not_allowed"},{"type":"alert_elephant_remote_to_local"},{"type":"alert_remote_to_local_insecure_proto"},{"type":"alert_tcp_syn_flood_attacker"},{"type":"alert_unexpected_smtp_server"},{"type":"alert_list_download_succeeded"},{"type":"alert_dropped_alerts"}]}
